synapse/rest/client/notifications.py

```diff
@@ -72,6 +72,12 @@ async def on_GET(self, request: SynapseRequest) -> Tuple[int, JsonDict]:
 
         next_token = None
 
+        serialize_options = SerializeEventConfig(
+            event_format=format_event_for_client_v2_without_room_id,
+            requester=requester,
+        )
+        now = self.clock.time_msec()
```

Comment:
I think leaving this inline would be fine, but don't feel strongly.

I find it a little confusing that in other files we push the `self.clock.time_msec()` inline and here we pull it out, it seems inconsistent, although not a huge dea.

Reply:
There are some files where events are serialized in a loop and the time is checked before the loop as well so 🤷

Reply:
That's done on purpose -- you want all the events to have the same time that they were serialized.

Reply:
So here, should I leave it like this or should I inline both the options and the time?

Reply:
We can leave them both here, that's fine! Theoretically slightly more efficient to not rebuild it, I guess? 🤷